torchvision/models/video/mvit.py

```diff
@@ -303,12 +453,18 @@ def __init__(
             temporal_size (int): The temporal size ``T`` of the input.
             block_setting (sequence of MSBlockConfig): The Network structure.
             residual_pool (bool): If True, use MViTv2 pooling residual connection.
+            residual_cls (bool): If True, the addition on the residual connection will include the class embedding.
+            rel_pos (bool): If True, use MViTv2's relative positional embeddings.
```

Comment:
NIt: `rel_pos_embed` possibly more clear, also `residual_cls` could be renamed to something like `residual_with_cls_embed`.

Reply:
Sounds good. I'll make the change.